<robot name="rov">
  <link name="auv_link">
    <visual>
      <origin rpy="1.5708 0 0" xyz="-0.35 0.3 -0.15"/>
      <geometry>
        <mesh filename="package://hit25_auv/meshes/rov_urdf.stl"
               scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="grey">
        <color rgba="0.5 0.5 0.5 1"/>
      </material>
    </visual>

    <collision>
      <!-- visual과 동일하게 맞춤 -->
      <origin rpy="1.5708 0 0" xyz="-0.35 0.3 -0.15"/>
      <geometry>
        <mesh filename="package://hit25_auv/meshes/rov_urdf.stl"
               scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>

    <inertial>
      <mass value="1.0"/>
      <origin xyz="0 0 0"/>
      <inertia ixx="0.1" ixy="0" ixz="0"
               iyy="0.1" iyz="0" izz="0.1"/>
    </inertial>
  </link>
</robot>
